flask/openbabel/src/server.py

```diff
@@ -60,6 +60,24 @@ def convert(output_format):
                                            out_options=out_options)
     return Response(data, mimetype=mime)
 
+@app.route('/atom_count', methods=['POST'])
+def atom_count():
+    """Get the number of atoms in a molecule
```

Comment:
I would concentrate on the bigger stuff, if we have a molecule Avogadro understands we can get atom count, molecular weight, etc. Leave it as you did it, but more focus on the unique things we need from Open Babel.

Reply:
Okay, sounds good. It's pretty close to using the Open Babel flask image for everything now - at which point we won't need to import it in mongochemserver anymore.

Reply:
Fast/tiny stuff like atom count, molecular weight should be done by Avogadro. You can load the molecule once, extract stats. I would say if we wanted to use OpenBabel for stuff like that we should avoid lots of tiny calls, and have it generate "stats" where it could tell us number of atoms, number of bonds, number of heavy elements, rotatable bonds, rings, weight etc. We should prefer one call over many loads of the molecule over a network endpoint. SVG and 3D coord generation are heavy, but even there we might tack on stat generation as a request to avoid many loads of the same molecule incurring a lot of overhead.

Reply:
@cryos In the latest commit, I changed the "atom_count" end point to be a "properties" end point, where it returns a json dict with more information than just the atom count. It currently only generates number of atoms, number of heavy atoms, formula, spaced formula, and mass, but we can add more in the future I think.

This means there are just two Open Babel end points currently: "/convert" and "/properties".